.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: test
```

Comment:
This is redundant - if you do want to separate it into two files, make sure to edit the publish workflow accordingly (as it also covers build steps).

Reply:
Tests have been merged into publish.yml